pkg/handler/handler.go

```diff
@@ -113,6 +115,15 @@ func New(l *zap.Logger, baseDir string) (*mux.Router, error) {
 	router.HandleFunc("/api/v1/namespaces/{namespace}/pods/{name}/log", func(w http.ResponseWriter, r *http.Request) {
 		vars := mux.Vars(r)
 		containerName := r.URL.Query().Get("container")
+		if containerName == "" {
+			containerName, err = getPodDefaultContainerName(baseDir, vars["namespace"], vars["name"])
```

Comment:
Please make this do a http request against localhost to fetch the pod rather than rebuilding the file finding logic. Also, it must return an error if there is more than one container